### DISCUSSION TITLE
help :)

### DISCUSSION THREAD

**Discussion by Susistar** [winstxnhdw/lc-hax#239] [Q&A]
fatal: not a git repository (or any of the parent directories): .git
i did git clone --recursive https://github.com/winstxnhdw/lc-hax.git in cmd prompt then i opened requirements.bat and it said that pls help

**Comment by winstxnhdw** [comment]
I've seen this so many times already. Fella first downloaded the repository as a zip even though no where in the README tells you to do so. Then they open up git bash which defaults to %USERPROFILE% and git clone, not realising git clone literally downloads the repository for you. Then they open up the zip file which isn't downloaded with git, try to run requirements.bat, which then prints this error.

**Reply by o777l** [reply]
I also cannot build it now and it was cloned properly. Some message about git repository

**Reply by winstxnhdw** [reply]
It'd be nice to know what that message is..

**Reply by Susistar** [reply]
add somewhere like: DO NOT DOWNLOAD THE .ZIP or smth but i figured it out and forgot to close this discussion

**Reply by winstxnhdw** [reply]
There used to be one but I removed it because people still kept downloading the zip anyways.
